app/importers/osp_data/osp_importer.rb

```diff
@@ -33,7 +33,7 @@ def format_and_populate
   private
 
   def is_good_date(row)
-    if row['submitted'].blank? && row['awarded'].present?
+    if (row['submitted'].blank? || row['submitted'] == '/  /') && row['awarded'].present?
```

Comment:
Do we still need to do this?  I thought we determined the `.format_date_fields` method below converts the '/   /'s to blanks.

Reply:
I remember it being finicky but I think it was due to excel formatting dates. Ill revert this one.